src/devices/lincukoo.ts

```diff
@@ -106,4 +110,91 @@ export const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: [{modelID: "TS0601", manufacturerName: "_TZE204_sndkanfr"}],
+        model: "SZLMR10",
+        vendor: "Lincukoo",
+        description: "Human Motion & Presence Sensor",
+        configure: tuya.configureMagicPacket,
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        onEvent: tuya.onEventSetTime,
+        exposes: [
+            e.presence(),
+            e.illuminance(),
+            e
+                .numeric("detection_distance", ea.STATE_SET)
+                .withValueMin(3)
+                .withValueMax(6)
+                .withValueStep(1.5)
+                .withUnit("m")
+                .withDescription("Maximum range"),
+            e.numeric("radar_sensitivity", ea.STATE_SET).withValueMin(0).withValueMax(9).withValueStep(1).withDescription("Sensitivity of the radar"),
+            e.numeric("fading_time", ea.STATE_SET).withValueMin(30).withValueMax(60).withValueStep(1).withDescription("Fading time").withUnit("s"),
+            e.binary("radar_switch", ea.STATE_SET, "ON", "OFF").withDescription("Radar switch"),
+            e.binary("indicator", ea.STATE_SET, "ON", "OFF").withDescription("LED indicator"),
+            e.enum("work_mode", ea.STATE_SET, ["pir_mode", "radar_mode", "combine_mode"]).withDescription("work mode of device"),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, "presence", tuya.valueConverter.trueFalse0],
+                [20, "illuminance", tuya.valueConverter.raw],
+                [13, "detection_distance", tuya.valueConverter.divideBy100],
+                [16, "radar_sensitivity", tuya.valueConverter.raw],
+                [103, "fading_time", tuya.valueConverter.raw],
+                [101, "indicator", tuya.valueConverter.onOff],
+                [102, "radar_switch", tuya.valueConverter.onOff],
+                [104, "work_mode", tuya.valueConverterBasic.lookup({pir_mode: tuya.enum(0), radar_mode: tuya.enum(1), combine_mode: tuya.enum(2)})],
+            ],
+        },
+    },
+
+    {
+        fingerprint: [{modelID: "TS0601", manufacturerName: "_TZE284_gw05grph"}],
+        model: "CZF02",
+        vendor: "Lincukoo",
+        description: "Finger Robot",
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        onEvent: tuya.onEventSetTime,
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.binary("switch", ea.STATE_SET, "ON", "OFF").withDescription("switch on/off"),
```

Comment:
```suggestion
            e.switch(),
```

and change `[1, "switch", tuya.valueConverter.onOff],` to `[1, "state", tuya.valueConverter.onOff],`

Reply:
done